### ISSUE TITLE
A move to closest way button

### ISSUE THREAD

**Issue by domdomegg** [mapillary/mapillary_issues#1081]
Looking at old threads say @dr2okevin's comment:
    i think a button in the edit view thats called "move to closest road/way" would be nice

I think this would be useful too, wanted to add it now during the construction of editor v2

**Comment by de-vries** [comment]
similar to #224

**Comment by domdomegg** [comment]
Yes, it is a duplicate of #97, #224, but I'd like to make sure it is included in editor v2.

**Comment by knikel** [comment]
@angelday: great feature to consider

**Comment by angelday** [comment]
_mapillary_issues is no longer an active repository._

_Let’s continue discussing features in our forum! We will import this issue as topic to the Mapillary Forum in the coming weeks. We’ll notify here once it’s been done. See you there!_
